src/components/CurrentItemsSection.jsx

```diff
@@ -0,0 +1,148 @@
+import { useEffect, useState } from "react";
+import { getItems } from "../utils/api";
+import ItemsContainer from "./ItemsContainer";
+import styles from "./ItemsSection.module.css";
+import { usePaginationByOffset } from "../hooks/usePaginationByOffset";
+import { useNavigate, useSearchParams } from "react-router-dom";
+import Pagination from "./Pagination";
+import { useIsLogin } from "../contexts/LoginStateContext";
+
+const LIST_TYPE = "current";
+const VISIBLE_PAGE_LENGTH = 5;
+
+const CurrentItemsSection = ({ pageSize }) => {
+  //prettier-ignore
+  const isLogin = useIsLogin();
+
+  const [searchParams, setSearchParams] = useSearchParams();
+  const initKeyword = searchParams.get("keyword") || "";
+  const [keyword, setKeyword] = useState(initKeyword);
```

Comment:
```suggestion
  const [searchParams, setSearchParams] = useSearchParams();
  
  // Search keyword
  const initKeyword = searchParams.get("keyword") || "";
  const [keyword, setKeyword] = useState(initKeyword);
```

정말 사소한 차이지만 관련있는 코드끼리 뭉쳐있을수있게끔 구분해주니 훨씬 의도가 더 잘 보이죠? 이렇게 뭉쳐있는 코드들의 용도를 설명하는 간단한 주석을 달아주는것도 좋은 습관입니다 :)